src/wasm-interpreter.h

```diff
@@ -3370,6 +3370,13 @@ class ModuleRunnerBase : public ExpressionRunner<SubType> {
     if (timeout.breaking()) {
       return timeout;
     }
+    // TODO: Add threads support.
+    //       For now, as there are no other threads, if there is no timeout then
+    //       error (this would hang forever), and if there is a timeout then
+    //       just report that we timed out.
+    if (timeout.getSingleValue().getInteger() < 0) {
+      return Flow(NONCONSTANT_FLOW);
+    }
```

Comment:
This should go after the `not equal` check if we decide to keep it.

Reply:
Thanks, fixed.